docs/src/real_world_example.rst

```diff
@@ -13,9 +13,9 @@ For the resin we use the commercial *Sartomer Arkema resin* which mainly consist
 As photo initiator we use TPO.
 With a refractometer we measure the refractive index :math:`n_\text{resin} = 1.4849`.
 We pour the resin into a cup. The photoinitiator is mixed into IPA. This is shaken until the TPO is dissolved.
-The ethanol with the TPO is mixed into the resin. It is mixed in a Kurabo Planetary Mixer for some minutes. 
+The IPA with the TPO is mixed into the resin. It is mixed in a Kurabo Planetary Mixer for some minutes. 
 In total, we mix roughly :math:`30\mathrm{mg}` of TPO into :math:`40\mathrm{mL}` of the resin.
-With a spectrometer, we determine the absorbance at our printing wavelength :math:`405\mathrm{nm}` to be :math:`A=0.2347/1\mathrm{cm}`. That means, :math:`\mu = 54.04\mathrm{m}^{-1}`-
+With a spectrometer, we determine the absorbance at our printing wavelength :math:`405\mathrm{nm}` to be :math:`A=0.2347/1\mathrm{cm}`. That means, :math:`\mu = 2.302\cdot A \approx 0.5404\mathrm{cm}^{-1}`-
```

Comment:
Lets write it as `log(10) * 10`

Reply:
Sure, I'll switch it to ln(10)*10, so we keep it in m^-1

Reply:
We can keep per `/cm`. In the `config.json` everything is in `/mm`. 

So maybe we also convert it also to per mm. Then it should be clear.